<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>autoreporting</class>
 <widget class="QWidget" name="autoreporting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1150</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Autoreporting</string>
  </property>
  <widget class="QCameraViewfinder" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>601</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_ok">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_cameralist">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="button_close">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>关闭摄像头</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cameralist">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>当前摄像头：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_grab">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GRAB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ba">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>90</y>
     <width>251</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_score">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>130</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>69</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_result">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>成绩</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hu">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>190</y>
     <width>54</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>环</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_history">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>300</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>历史成绩</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>胸环靶</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>环形靶</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>摄像头预览</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>结果预览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCameraViewfinder</class>
   <extends>QWidget</extends>
   <header>qcameraviewfinder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
